.github/workflows/build.yaml

```diff
@@ -0,0 +1,73 @@
+name: Build for slsa
+on:
+  pull_request:
+  push:
+    tags:
+      - 'v*'
+      
+jobs:
+  build:
+    runs-on: "ubuntu-latest"
+    outputs:
+      hashes: ${{ steps.hash.outputs.hashes }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+      - name: Generate checksum
+        id: hash
+        run: |
+          # Install 'build' and create sdist & wheel
+          python -m pip install build
+          python -m ensurepip --upgrade
+          python -m pip install -r requirements.txt
+          python -m build
+
+          # Gather hashes of all files
+          set -euo pipefail
+          cd dist && echo "hashes=$(sha256sum * | base64 -w0)" >> $GITHUB_OUTPUT
+          cd ..
+      - uses: actions/upload-artifact@v4
+        with:
+          path: dist/
+      - name: Create Release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref_name }}
+          draft: false
+          prerelease: false
+          args: release --clean
```

Comment:
I think that we should let the team do the releases manually for now, they can always automate it later